examples/README.md

```diff
@@ -17,6 +17,7 @@ pip install -r https://raw.githubusercontent.com/Project-MONAI/MONAI/master/requ
 ```
 
 ### 2. List of examples
+#### [A brief guide to MONAI for PyTorch users (Colab)](https://colab.research.google.com/drive/1boqy7ENpKrqaJoxFlbHIBnIODAs1Ih1T)
```

Comment:
1. Suggest also add this link to our main README directly in "Quick Start" guide, what do you think?
![image](https://user-images.githubusercontent.com/27533405/86312196-c107a580-bc54-11ea-89f2-43c8a261a959.png)
2. Suggest also adding the previous `MedNIST Colab notebook` to this README and the quick start of main README.

Reply:
sure I changed the link and I'll update the notebook

Reply:
Cool, let's merge this PR when your Colab notebook is finalized.
Thanks.